de-lib/src/main/java/org/iplantc/de/client/models/UserSettings.java

```diff
@@ -271,6 +272,20 @@ public Splittable asSplittable() {
         return ret;
     }
 
+    public void useDefaultValues(UserInfo instance) {
+        JSONObject defaults = new JSONObject();
+        JSONObject defaultHomeDir = new JSONObject();
+
+        JSONString id = new JSONString(instance.getHomePath());
+        JSONString path = new JSONString(instance.getHomePath());
```

Comment:
Keep in mind that this `getHomePath` method can currently return `null`, if something goes wrong parsing the user info JSON in the `BootstrapCallback`.

I'm guessing that the `getHomePath` method will still be updated in another PR to return a default in this case? We will also be updating the bootstrap endpoint to return an error message instead of the user's home path when `data-info` is down.